xds/internal/client/load/store.go

```diff
@@ -24,8 +24,58 @@ import (
 
 const negativeOneUInt64 = ^uint64(0)
 
-// Store is a repository for LB policy implementations to report store load
-// data. It is safe for concurrent use.
+// A pair of cluster and service name. The same cluster can be used by multiple
+// services, and one service can use multiple clusters. So we need a pair with
+// both name to accurately indicate where the load belongs.
+type clusterAndServiceNames struct {
+	cluster string
+	service string
+}
+
+type Store struct {
+	clusters sync.Map // map[clusterAndServiceNames]*rpcCountData
```

Comment:
Could we please not add more `sync.Map`s when there is no need for it.

Reply:
Done.